Mlem/Views/Shared/Accounts/Instance Picker View.swift

```diff
@@ -15,45 +15,72 @@ struct InstancePickerView: View {
     
     @Binding var selectedInstance: InstanceMetadata?
     
-    @State var instances: [InstanceMetadata]?
+    @State private var query: String = ""
+    @State private var instances: [InstanceMetadata]?
     
     /// Instances currently accepting new users
-    var filteredInstances: [InstanceMetadata]? {
+    var filteredInstances: ArraySlice<InstanceMetadata>? {
         instances?
-            .filter { instance in
-                instance.newUsers
-            }
+            .sorted(by: { $0.users > $1.users }) // remote source is already sorted by user count but that may change...
+            .filter(\.newUsers) // restrict the list to instances who are accepting new users
```

Comment:
While we're filtering, can we exclude NSFW instances from the default list as well? As far as I can tell the only two in the list are lemmynsfw and pornlemmy--both of them conveniently have "nsfw" in their version number, so we could also use that as a filter criterion.